catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ImportCommand.java

```diff
@@ -0,0 +1,242 @@
+package org.codice.ddf.commands.catalog;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Paths;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.concurrent.atomic.AtomicReferenceFieldUpdater;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+import org.apache.commons.io.FilenameUtils;
+import org.apache.karaf.shell.api.action.Argument;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.Option;
+import org.apache.karaf.shell.api.action.lifecycle.Reference;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.codice.ddf.catalog.transformer.zip.ZipValidator;
+import org.codice.ddf.commands.util.CatalogCommandRuntimeException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.io.ByteSource;
+
+import ddf.catalog.content.StorageProvider;
+import ddf.catalog.content.data.ContentItem;
+import ddf.catalog.content.data.impl.ContentItemImpl;
+import ddf.catalog.content.operation.impl.CreateStorageRequestImpl;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.operation.impl.CreateRequestImpl;
+import ddf.catalog.transform.CatalogTransformerException;
+import ddf.catalog.transform.InputTransformer;
+import ddf.security.common.audit.SecurityLogger;
+
+@Service
+@Command(scope = CatalogCommands.NAMESPACE, name = "import", description = "Imports Metacards and history into the current Catalog")
+public class ImportCommand extends CatalogCommands {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ImportCommand.class);
+
+    private static final int ID = 2;
+
+    private static final int TYPE = 3;
+
+    private static final int NAME = 4;
+
+    private static final int DERIVED_NAME = 5;
+
+    @Reference
+    private StorageProvider storageProvider;
+
+    @Argument(name = "Import File", description = "The file to import", index = 0, multiValued = false, required = true)
+    String importFile;
+
+    @Option(name = "--UNSAFE-NO-VERIFY", required = false, multiValued = false, description = "Exports the data but does NOT sign the resulting zip file. "
```

Comment:
I would suggest changing this to something like `--skip-signature-verification`.  Also the first sentence of the description needs to be updated for import instead of export.  Also include a `WARNING: ` before the second sentence.